lms/djangoapps/teams/static/teams/js/spec/views/team_profile_header_actions_spec.js

```diff
@@ -0,0 +1,254 @@
+define([
```

Comment:
I've renamed this file with others but i don't know why only this file is different from others. Let me give it a look.